modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -789,9 +789,9 @@ export const console: ConsoleNS = {
                                     "by navigating to <3>Attribute Mappings.</3>",
                                 attributeComponentHintAlt: "Manage the user attributes you want to share with this" +
                                     " application.",
-                                mandatoryAttributeHint: "Mark which attributes are mandatory for a user to share" +
-                                    " with the application. When logging in, Asgardeo prompts the user to enter the" +
-                                    " attribute value if it is not already provided in the user's profile.",
+                                mandatoryAttributeHint: "Mark which user attributes are mandatory to be shared " +
+                                    "with the application. At login, Asgardeo prompts the user to enter these " +
```

Comment:
We shouldn't mention the product name, should we?

Reply:
I will update the code to get it from the config